src/components/search/SearchHeader.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useContext } from 'react';
+import { AppContext } from '@edx/frontend-platform/react';
+
+import SearchBox from './SearchBox';
+import SearchFilters from './SearchFilters';
+import CurrentRefinements from './CurrentRefinements';
+
+const SearchHeader = () => {
+  const { enterpriseConfig: { branding } } = useContext(AppContext);
+
+  return (
+    <div style={{ backgroundColor: branding.search.backgroundColor }}>
```

Comment:
Replace inline style with `.bg-brand-primary` once https://github.com/edx/frontend-app-learner-portal-enterprise/pull/36 is merged.